src/Mediator.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { BrowserRouter as Router, Routes,Route, Link} from "react-router-dom";
+import App from './App';
+import Home from './Home';
+
+
+function Mediator() {
+    return (
+        <div>
+            <Router>
+ 
```

Comment:
you have used .tsx for some files and .jsx for some files . Convert all the files in .tsx and add typescript in it

Reply:
Just 1-2 file is jsx was trying to see if both react and typescript works together well as in most projects they go hand in hand